mllib-dal/src/main/native/oneapi/dal/HomogenTableImpl.cpp

```diff
@@ -293,6 +293,47 @@ JNIEXPORT jlong JNICALL Java_com_intel_oneapi_dal_table_HomogenTableImpl_lInit(
      return (jlong)tablePtr.get();
 }
 
+/*
+ * Class:     com_intel_oneapi_dal_table_HomogenTableImpl
+ * Method:    Init
+ * Signature: (JJJIII)J
+ */
+JNIEXPORT jlong JNICALL Java_com_intel_oneapi_dal_table_HomogenTableImpl_dPtrInit(
+    JNIEnv *env, jobject, jlong cRowCount, jlong cColCount, jlong dataPtr, jint cLayout, jint computeDeviceOrdinal) {
+    printf("HomogenTable dPtrInit\n");
```

Comment:
could you remove debug print

Reply:
Will fix it in PR(https://github.com/oap-project/oap-mllib/pull/297)